distributednetworkmanager/src/main/java/com/gruppo1/distributednetworkmanager/KBucket.java

```diff
@@ -74,4 +74,11 @@ public Node getOldest() {
             return elements[0];
         return null;
     }
+
+    /**
+     * @return number of nodes in the bucket
+     */
+    public int size(){
```

Comment:
Did you test this? I've written this class so I'm quite sure `last` should always be up to date, but I would not be 100% sure xD

Reply:
It should be ok